custom_generator.js

```diff
@@ -0,0 +1,856 @@
+/**
+ * Custom bingo generator, taking into account dynamic difficulty.
+ */
+
+ var bingo_list_custom = {
```

Comment:
`bingo_list_custom` is in its own file, so it should be removed from here

Reply:
That's my plan for before I mark as ready. Right now having everything in one file makes it easier to debug locally.